documentation/docs/07-a-options.md

```diff
@@ -43,3 +43,10 @@ Adding a `rel=external` attribute to a link...
 ```
 
 ...will trigger a browser navigation when the link is clicked.
+The same behaviour applies if you set the `rel` attribute to either `noopener` or `noreferrer` as those values imply external navigation.
+
+It's also possible to specify more than one value for the `rel` attribute by seperating values with a whitespace:
```

Comment:
I don't think we need to document this. I'd consider it a bug that it didn't work before, but we don't need to document every bug fix

Reply:
Makes sense, I'm unsure if the explicit handling of `noopener` and `noreferrer` should be omitted in the implementation as well, as it would also be possible to write `rel="external noreferrer`. My thinking was that links that get created from 3rd party libraries somewhere on the page might not explicitely have an `external` value set, but might have either `noopener` or `noreferrer` set

Reply:
I'm not sure what the best behavior would be for those other attributes. I'd want the other maintainers to weigh in. But since they've been removed I'll go ahead and merge this